pkgs/applications/misc/waybar/default.nix

```diff
@@ -50,7 +51,7 @@ stdenv.mkDerivation rec {
   strictDeps = false;
 
   buildInputs = with lib;
-    [ wayland wlroots gtkmm3 libsigcxx jsoncpp spdlog gtk-layer-shell howard-hinnant-date libxkbcommon ]
+    [ wayland wlroots gtkmm3 libsigcxx jsoncpp spdlog gtk-layer-shell howard-hinnant-date libxkbcommon catch2 ]
```

Comment:
Double check but I think this should be a checkInput; it's only used for tests

Reply:
That's fair; I only just realised it wasn't working before (see the message to b38e6cbbb3a1e0c0a8ae995efe337deb19e5f8f5) because checkInputs only get added if doCheck is true.

The package doesn't compile by default at all because building the tests is enabled by default, which forces a hard requirement on catch2. I've added a new flag to the package to allow turning the tests off which also passes the necessary flag to meson.